app/views/settings/identity_proofs/_proof.html.haml

```diff
@@ -1,14 +1,16 @@
-
 %tr
-  %td{ style: 'font-size: 16px; text-align: center; vertical-align: middle' }
+  %td= proof.provider
+  %td.account
     - provider_pic_url = proof.remote_profile_pic_url || default_provider_pic_url(proof.provider)
-    .avatar= image_tag provider_pic_url, width: 50
-  %td{ style: 'font-size: 16px; text-align: center; vertical-align: middle' }
-    %strong="#{proof.provider_username} "
-    ="on #{proof.provider}"
-  %td{style: 'vertical-align: middle'}
-    %div
-      %center
-        -if proof.keybase?
-          =I18n.t('account_identity_proofs.keybase.proof_status', path: proof.remote_url).html_safe
-          =image_tag proof.status_img_url
+    = link_to proof.profile_url do
+      = image_tag provider_pic_url, size: 40, class: :account__avatar
+      = proof.provider_username
+  %td
+    - status = proof.is_valid ? 'Valid' : 'Invalid'
```

Comment:
this is supposed to do a live lookup for the status right now on the provider (keybase). or at least, that's what we had talked about before. that's why it was doing the SVG thing. that image was decided in real time by keybase as the thing to show for this proof

Reply:
Can we get a boolean response instead of the image? Cécile and I don't think we should use status images for this.